opentreemap/treemap/decorators.py

```diff
@@ -168,12 +168,9 @@ def run_and_catch_validations(*args, **kwargs):
             return req(*args, **kwargs)
         except ValidationError as e:
             if hasattr(e, 'message_dict'):
+                message_dict['fieldErrors'] = e.message_dict
                 message_dict['globalErrors'] = [_(
                     'One or more of the specified values are invalid.')]
-                if 'globalErrors' in e.message_dict:
-                    message_dict['globalErrors'] += \
-                        e.message_dict.pop('globalErrors')
-                message_dict['fieldErrors'] = e.message_dict
```

Comment:
That does look like it may be a keeper. @RobinIsTheBird can you confirm this? Would you want to include anything else in a commit that restores these lines?